src/custom/state/sentry/updater/index.ts

```diff
@@ -0,0 +1,49 @@
+import { useEffect } from 'react'
+import * as Sentry from '@sentry/browser'
+
+import useIsWindowVisible from 'hooks/useIsWindowVisible'
+import { useSwapState } from 'state/swap/hooks'
+import { SupportedChainId } from 'constants/chains'
+import { useCurrency } from 'hooks/Tokens'
+import { useWalletInfo } from 'hooks/useWalletInfo'
+
+export default function Updater(): null {
+  const { account, chainId, walletName, isSupportedWallet } = useWalletInfo()
+  const windowVisible = useIsWindowVisible()
+
+  const {
+    INPUT: { currencyId: sellTokenAddress },
+    OUTPUT: { currencyId: buyTokenAddress },
+  } = useSwapState()
+
+  const sellCurrency = useCurrency(sellTokenAddress)
+  const buyCurrency = useCurrency(buyTokenAddress)
+
+  // create sentry context based on "main" parameters
+  useEffect(() => {
+    if (windowVisible) {
+      Sentry.setContext('user', {
+        userAddress: account || 'DISCONNECTED',
```

Comment:
David, if possible validate with legal about this

Reply:
that will take a while

Reply:
i see u removed. pls makes sure is ok and re-enable if possible